src/ext/java/org/opentripplanner/ext/siri/SiriFuzzyTripMatcher.java

```diff
@@ -191,6 +192,16 @@ public FeedScopedId getTripId(String vehicleJourney) {
     Trip trip = vehicleJourneyTripCache.get(vehicleJourney);
     if (trip != null) {
       return trip.getId();
+    } else {
+      //Attempt to find trip using datedServiceJourneys
+      for (String feedId : routingService.getFeedIds()) {
```

Comment:
We'd want to push in a specific `feedId` to use here from the individual `GraphUpdater`, but I see that this is done like this already in other places. Would it be a too big job to do this refactoring now?